src/Header/AbstractAccept.php

```diff
@@ -438,10 +438,12 @@ protected function sortFieldValueParts()
             //@todo count number of dots in case of type==application in subtype
 
             // So far they're still the same. Longest string length may be more specific
-            if (strlen($a->raw) == strlen($b->raw)) {
+            $aLength = strlen($a->raw);
+            $bLength = strlen($b->raw);
+            if ($aLength === $bLength) {
```

Comment:
Spaceship operator: only case where I ever suggested its usage.

Reply:
Can't use it here; this component still supports PHP 5.6.